src/components/PullToRefresh/PullToRefresh.tsx

```diff
@@ -0,0 +1,22 @@
+import globalStyles from 'constants/styles';
+import React from 'react';
+import {RefreshControl, ScrollView} from 'react-native';
+
+interface PullToRefreshProps {
+  children: React.ReactChild;
+  profileLoading?: boolean;
+  refreshing: boolean;
+  onRefresh: () => Promise<any>;
+}
+
+export default function PullToRefresh(props: PullToRefreshProps) {
```

Comment:
For the native part, this component is useless, and it needs to return an empty fragment.
This component is only web